web/cobrands/greenwich/layout.scss

```diff
@@ -257,3 +374,156 @@ body.twothirdswidthpage .content .sticky-sidebar {
3.62a2.58%2C2.58%2C0%2C0%2C1%2C2.94.4%2C9.69%2C9.69%2C0%2C0%2C1%2C2%2C2.33%2C9.07%2C9.07%2C0%2C0%2C0-1.74%2C2.59%2C8.68%2C8.68%2C0%2C0%2C0-.23%2C3.25%2C8.88%2C8.88%2C0%2C0%2C1-3.76.42Zm8.49%2C7.74c-.73%2C1.2-1.6%2C1.66-2.37%2C1.4s-2.41-1.92-3-2.32-2.89-1.63-3.05-2.41a2.61%2C2.61%2C0%2C0%2C1%2C1.25-2.7A9.82%2C9.82%2C0%2C0%2C1%2C155.49%2C33a8.94%2C8.94%2C0%2C0%2C0%2C2%2C2.41%2C8.35%2C8.35%2C0%2C0%2C0%2C3%2C1.17%2C8.83%2C8.83%2C0%2C0%2C1-.68%2C3.69Zm1.6-6.18a1.29%2C1.29%2C0%2C0%2C1%2C0%2C.48.86.86%2C0%2C0%2C1-.29.31%2C1%2C1%2C0%2C0%2C1-.3-.33%2C1%2C1%2C0%2C0%2C1%2C0-.49h0s-.5%2C1.06-2.21-.06a1.4%2C1.4%2C0%2C0%2C1-.69-1.87l0%2C0a1.32%2C1.32%2C0%2C0%2C1-.48.09.87.87%2C0%2C0%2C1-.37-.19.91.91%2C0%2C0%2C1%2C.24-.37.86.86%2C0%2C0%2C1%2C.46-.17s-1.2-.32-.58-2.14c.55-1.58%2C1.63-1.2%2C1.63-1.2a1.29%2C1.29%2C0%2C0%2C1-.26-.42.86.86%2C0%2C0%2C1%2C.05-.42%2C1.13%2C1.13%2C0%2C0%2C1%2C.44.1.93.93%2C0%2C0%2C1%2C.31.37s0-1.24%2C1.76-1.24%2C1.72%2C1.25%2C1.72%2C1.25a1.87%2C1.87%2C0%2C0%2C1%2C.36-.43%2C1%2C1%2C0%2C0%2C1%2C.42-.08%2C1.09%2C1.09%2C0%2C0%2C1%2C0%2C.44.87.87%2C0%2C0%2C1-.27.41s1.12-.36%2C1.71%2C1.34a1.63%2C1.63%2C0%2C0%2C1-.53%2C2%2C1.27%2C1.27%2C0%2C0%2C1%2C.45.19.91.91%2C0%2C0%2C1%2C.21.37.77.77%2C0%2C0%2C1-.89.06s.68%2C1-.81%2C2-2.12%2C0-2.12%2C0Zm9.41%2C3.18c-.48.66-2.55%2C1.73-3.1%2C2.15s-2.41%2C2.28-3.21%2C2.2a2.56%2C2.56%2C0%2C0%2C1-2.2-2%2C9.88%2C9.88%2C0%2C0%2C1-.37-3.06%2C8.93%2C8.93%2C0%2C0%2C0%2C2.9-1.17%2C8.74%2C8.74%2C0%2C0%2C0%2C2-2.56%2C9%2C9%2C0%2C0%2C1%2C3.33%2C1.75c.92%2C1.05%2C1.1%2C2%2C.63%2C2.68Zm-.73-4.72a9.7%2C9.7%2C0%2C0%2C1-3-.73%2C9%2C9%2C0%2C0%2C0-.08-3.11%2C8.49%2C8.49%2C0%2C0%2C0-1.69-2.79%2C8.82%2C8.82%2C0%2C0%2C1%2C2.79-2.51c1.32-.5%2C2.29-.33%2C2.74.34s.73%2C3%2C.93%2C3.66%2C1.3%2C3.05.95%2C3.77a2.6%2C2.6%2C0%2C0%2C1-2.66%2C1.36ZM119.55%2C63.33a5.26%2C5.26%2C0%2C0%2C0-.08-.89%2C2.05%2C2.05%2C0%2C0%2C0-.27-.75%2C1.46%2C1.46%2C0%2C0%2C0-.52-.5%2C1.61%2C1.61%2C0%2C0%2C0-.83-.19%2C1.54%2C1.54%2C0%2C0%2C0-.56.1%2C5.28%2C5.28%2C0%2C0%2C0-.53.27%2C4.29%2C4.29%2C0%2C0%2C0-.48.36c-.15.15-.29.27-.42.38l-1.11%2C5a.85.85%2C0%2C0%2C0%2C.15.37%2C1.22%2C1.22%2C0%2C0%2C0%2C.32.36%2C1.77%2C1.77%2C0%2C0%2C0%2C1.17.39%2C2.19%2C2.19%2C0%2C0%2C0%2C1.24-.39%2C3.4%2C3.4%2C0%2C0%2C0%2C1-1%2C5.42%2C5.42%2C0%2C0%2C0%2C.69-1.6%2C7.44%2C7.44%2C0%2C0%2C0%2C.23-1.82m1.71%2C0a5.56%2C5.56%2C0%2C0%2C1-.4%2C2.13%2C6.08%2C6.08%2C0%2C0%2C1-1.06%2C1.76%2C5.36%2C5.36%2C0%2C0%2C1-1.53%2C1.18%2C3.92%2C3.92%2C0%2C0%2C1-1.81.43%2C5.91%2C5.91%2C0%2C0%2C1-1.26-.12%2C3.89%2C3.89%2C0%2C0%2C1-.94-.33%2C2.85%2C2.85%2C0%2C0%2C1-.65-.49%2C3.9%2C3.9%2C0%2C0%2C1-.46-.61l2.16-9.47a.51.51%2C0%2C0%2C1%2C.07-.33%2C1.41%2C1.41%2C0%2C0%2C0%2C0-.29%2C1%2C1%2C0%2C0%2C0-.1-.46.71.71%2C0%2C0%2C0-.29-.25%2C1.65%2C1.65%2C0%2C0%2C0-.44-.1l-.55-.06.12-.5%2C3.1-.18.08.12L116%2C61.41h0a5.49%2C5.49%2C0%2C0%2C1%2C1.32-.92%2C2.56%2C2.56%2C0%2C0%2C1%2C1.12-.26%2C2.48%2C2.48%2C0%2C0%2C1%2C2%2C.86%2C3.55%2C3.55%2C0%2C0%2C1%2C.72%2C2.33%22%20fill%3D%22%23ffffff%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E);
```

Comment:
Embedding very small images/icons into the Sass files is ok, but once we get this big (this is a white-on-transparent version of the Greenwich logo) we’d usually just save it as a file and reference it by URL like a normal image.